yapapi/executor/ctx.py

```diff
@@ -201,16 +217,23 @@ def send_file(self, src_path: str, dst_path: str):
         self.__prepare()
         self._pending_steps.append(_SendFile(self._storage, src_path, dst_path))
 
-    def run(self, cmd: str, *args: Iterable[str], env: Optional[Dict[str, str]] = None):
+    def run(
+        self, cmd: str, *args: Iterable[str], env: Optional[Dict[str, str]] = None,
```

Comment:
Those were removed on purpose (as you can see in https://github.com/golemfactory/yapapi/pull/35/files#diff-2b03ec5372ff1463a529219f90583a4ba2e59a3d732924abd6fc8f20987ecc55R220-R227). The API is not exposed since it hasn't been decided yet on how to handle different output capture modes.